app/middlewares/token/checkToken.js

```diff
@@ -0,0 +1,16 @@
+import jwt from 'jsonwebtoken';
+import { db_config } from '../../config/db_config';
+
+export const checkToken = (req, res, next) => {
+  if (req.path === '/login') return next();

```

Comment:
It will be better to move all your route paths to const as they are started repeating

Reply:
You are right, fixed it